src/style.css

```diff
@@ -1,3 +1,77 @@
+*{
+  box-sizing: border-box;
+}
+
+a{
+  text-decoration: none;
+}
+
+:root{
+  --blue-color: #00ACDC;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.header{
+  width: 100%;
+  display: flex;
+  align-items: center;
+  padding: 0 50px;
+  justify-content: space-between;
+}
+
+img{
+  display: block;
+}
+
+.nav{
+  display: flex;
+  height: 60px;
+  align-items: center;
+}
```

Comment:
redundant
```suggestion
```

Reply:
isn't fixed. delete these rules for `.nav` (for your case nav shouldn't have styles)